conf/psc/queue.yml

```diff
@@ -0,0 +1,14 @@
+queue-maintainers:
+  members:
+    - guewen
+    - lmignon
+  name: Asynchronous job queue maintainers
+  representatives:
+    - simahawk
+    - sbidoul
+    - gurneyalex
```

Comment:
@etobella what does this reprensentative section mean here, versus the members of the corresponding psc-representative group?

Reply:
Isn't this specified here? https://odoo-community.org/about/psc-guide

Reply:
Oh, understood that is a technical question of the yaml file.

Reply:
representatives are maintainers of the team
https://github.com/OCA/repo-maintainer/blob/44446eb1694db5b9699d1a3af0ea4db13b14beed/oca_repo_maintainer/tools/manager.py#L126-L129

Reply:
So do we still need two groups `queue-maintainers` + `queue-maintainers-psc-representative` ?

Reply:
Actually I still don't understant :) I thought the role would be for the group, not for the group members.